executors/sshexec/volume.go

```diff
@@ -142,6 +142,25 @@ func (s *SshExecutor) VolumeDestroy(host string, volume string) error {
 	return nil
 }
 
+func (s *SshExecutor) VolumeStatusDetail(host string, volume string) (string, error) {
```

Comment:
This functionality is also implemented in PR #710 , I suggest you wait for that patch to be merged to avoid conflicts.